sdk/runtime/metrics.go

```diff
@@ -0,0 +1,61 @@
+package runtime
+
+import (
+	"io"
+	"net"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+// MustExportPrometheus starts an HTTP server with the Prometheus handler.
+// It starts on a random open port and returns the http address.
+func (re *RunEnv) MustExportPrometheus() string {
+	listener, err := net.Listen("tcp", ":0")
+	if err != nil {
+		panic(err)
+	}
+
+	go http.Serve(listener, promhttp.Handler())
+
+	port := strconv.Itoa(listener.Addr().(*net.TCPAddr).Port)
```

Comment:
You could probably just return the listener, or the listener.Addr().String() and not have to do any conversions.

Reply:
Probably best to return the listener, so that we can close it, when we are done with exposing metrics on shutdown?

Reply:
Well, we want the HTTP address, don't we? We'll need to convert it anyhow to use with `http.Get`.